src/otaclient/boot_control/protocol.py

```diff
@@ -37,6 +37,11 @@ def get_booted_ota_status(self) -> OTAStatus:
     def get_standby_slot_path(self) -> Path:
         """Get the Path points to the standby slot mount point."""
 
+    @property
+    @abstractmethod
+    def standby_slot_dev(self) -> Path:
+        """The device of the standby slot."""
+
```

Comment:
When checking whether we can use `inplace` or not, we need to direct access to the standby slot's device. Let boot controller implementation exposes this information.